### DISCUSSION TITLE
Adding Configurable Pipeline for the Analysis of Connectomes (CPAC)

### DISCUSSION THREAD

**Discussion by joshleelab** [neurodesk/neurodesk.github.io-hugo#378] [New Tool]
Have you Considered adding CPAC: Configurable Pipeline for the Analysis of Connectomes
https://fcp-indi.github.io/

**Comment by stebo85** [comment]
Dear @nimzodisaster,

That would be great :) Would you like to propose a pull request that adds it? It would be quite similar to fMRIprep and should be straight forward to add: https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/fmriprep/build.sh

Thank you
Steffen

**Reply by stebo85** [reply]
Dear @nimzodisaster - are you adding CPAC to Neurodesk? Do you need any help?

**Comment by joshleelab** [comment]
I've been swamped...and I just started looking into neurodesk. I'm also not
on the cpac team, so maybe they should??

**Reply by stebo85** [reply]
yes, developers are often a good choice - they know their tool best :) Can you contact them and ask if they are keen to add their tool?

**Comment by stebo85** [comment]
Dear @nimzodisaster - are you still working on this? Do you need any help?

**Comment by stebo85** [comment]
cpac is now available in neurodesk:
<img width="676" height="262" alt="image" src="https://github.com/user-attachments/assets/980afdcd-69b3-4cd2-bbf2-a32108f438a1" />
